apps/frontend/src/pages/management/libs/components/groups-tab/groups-tab.tsx

```diff
@@ -213,41 +196,14 @@ const GroupsTab: React.FC = () => {
 					</Table>
 				</div>
 			</div>
-			<EditModal
+			<EditGroupModal
+				group={currentGroup ?? DEFAULT_GROUP}
```

Comment:
```suggestion
     {currentGroup &&
     			<EditGroupModal
		      		group={currentGroup}
```
maybe, it is better?

Reply:
Conditional rendering removes popup animation, but I need to keep it.